DB/db.sql

```diff
@@ -32,21 +32,11 @@ CREATE TABLE opera (
       nome VARCHAR(30) NOT NULL,
       descrizione VARCHAR(150) NOT NULL,
       prezzo DECIMAL(10,2) NOT NULL,
-      possessore VARCHAR(30),
+      possessore VARCHAR(30) NOT NULL,
```

Comment:
```suggestion
      possessore VARCHAR(30) NOT NULL DEFAULT 'admin',
```
Da aggiungere visto che di default, quando vengono caricate, gli nft sono dell'azienda/sito cioè dell'admin